react-app/src/app/screens/User/components/Points/components/Rules/index.js

```diff
@@ -1,19 +1,15 @@
-import React, { Component } from 'react';
+import React from 'react';
 
 import { POINTS_RULES } from '../../../../../../constants';
 
-class Rules extends Component {
-  render() {
-    return (
-      <ul>
-        {POINTS_RULES.map((rule, i) => (
-          <li key={i.toString()}>
-            Obtain <b>{rule.points} points</b> winning with {rule.moves} moves
-          </li>
-        ))}
-      </ul>
-    );
-  }
-}
+const Rules = () => (
```

Comment:
We dont use arrow functions to define components.
For stateless comps we used classic functions definitions, and we leave arrow for functions and methods